packages/core/lib/debug/interpreter.js

```diff
@@ -503,9 +503,14 @@ class DebugInterpreter {
           temporaryPrintouts.add(location);
         }
         if (this.session.view(selectors.session.status.loaded)) {
-          this.printer.printInstruction(temporaryPrintouts);
-          this.printer.printFile();
-          this.printer.printState();
+          if (this.session.view(trace.steps).length > 0) {
+            this.printer.printInstruction(temporaryPrintouts);
+            this.printer.printFile();
+            this.printer.printState();
+          } else {
+            //if there are no trace steps, let's just print a warning message
+            this.printer.print("No trace steps to inspect.");
```

Comment:
Shouldn't this call that method you made?

```suggestion
            this.printer.warnIfNoSteps();
```

Reply:
No, it's deliberately a different message.  The `warnIfNoSteps` here wouldn't make sense here.  A warning is printed in *addition* to something, whereas this is in *place* of the usual printout.

Reply:
Ah, so then it should probably be more in-your-face. Is this one an error?

Reply:
I don't think so?  There's no error here, there just aren't any trace steps to inspect.  Remember the user has to deliberately hit `p` to see this.  They've already been warned there are no trace steps.  If they try to hit `p` anyway, well, they get a message telling them there's nothing to see.  It can be marked "error" if you think that's appropriate, I guess?  But to my mind it's not much different than the "no transaction loaded" messages users will get if they try to do things with no transaction loaded.  (Or should those be marked error too?)

Reply:
Ah I see. OK then this is probably be fine